src/library_dylink.js

```diff
@@ -309,14 +314,15 @@ var LibraryDylink = {
         return dynCall(sig, arguments[0], Array.prototype.slice.call(arguments, 1));
       } catch(e) {
         stackRestore(sp);
-        // Exceptions thrown from C++ exception will be integer numbers.
-        // longjmp will throw the number Infinity. Re-throw other types of
-        // exceptions using a compact and fast check.
+        // Exceptions thrown from C++ will be an integer number and longjmp will throw
+        // the number Infinity. Use the compact and fast "e !== e+0" test to check if
+        // e was not a Number.
```

Comment:
```suggestion
        // Exceptions thrown from C++ will be an integer number and longjmp will
        // throw the number Infinity. Use the compact and fast "e !== e+0" test
        // to check if e was not a Number, and rethrow it if not.
```
- Wrapped the comment to 80 column. (We don't seem to do 80 column wrapping strictly in Emscripten, but at least comments in this file look wrapped to 80 cols)
- Added "and rethrow it if not" at the end, to preserve the original comment's info